lib/scala/pkg/src/main/scala/org/enso/pkg/ComponentGroup.scala

```diff
@@ -0,0 +1,316 @@
+package org.enso.pkg
+
+import io.circe._
+import io.circe.syntax._
+
+/** The description of component groups provided by the package.
+  *
+  * @param newGroups the list of component groups provided by the package
+  * @param extendedGroups the list of component groups that this package extends
+  */
+case class ComponentGroups(
+  newGroups: List[ComponentGroup],
+  extendedGroups: List[ExtendedComponentGroup]
+)
+object ComponentGroups {
+
+  /** An empty component groups. */
```

Comment:
This is a very minor thing but something doesn't seem right here.

```suggestion
  /** Empty component groups. */
```

```suggestion
  /** An empty component group. */
```